docs/.vuepress/plugins/dump-docs-data/docs-versions.js

```diff
@@ -46,15 +51,29 @@ async function readFromGitHub() {
   }
 
   const tagsSet = new Set();
+  const minorsToPatches = new Map();
 
   releases.data
     .map(item => item.tag_name)
     .sort((a, b) => semver.rcompare(a, b))
-    .forEach(tag => tagsSet.add(`${semver.parse(tag).major}.${semver.parse(tag).minor}`));
+    .forEach((tag) => {
+      const minorVersion = `${semver.parse(tag).major}.${semver.parse(tag).minor}`;
+      const patchVersion = `${minorVersion}.${semver.parse(tag).patch}`;
 
-  const tags = Array.from(tagsSet);
+      tagsSet.add(minorVersion);
+
+      if (minorsToPatches.has(minorVersion)) {
+        const uniquePathes = Array.from(new Set(minorsToPatches.get(minorVersion)).add(patchVersion));
+
+        minorsToPatches.set(minorVersion, uniquePathes);
```

Comment:
```suggestion
        const uniquePaths = Array.from(new Set(minorsToPatches.get(minorVersion)).add(patchVersion));

        minorsToPatches.set(minorVersion, uniquePaths);
```

Reply:
Changed within https://github.com/handsontable/handsontable/pull/10079/commits/e539c2a3a296ae4735d921341bed747ab79092d7 from `uniquePathes` to `uniquePatches`.